pkg/controller/discovery/web/web.go

```diff
@@ -350,7 +350,7 @@ func (h *BaseHandler) allow(sar auth.SelfSubjectAccessReview) int {
 		Log.Trace(err)
 		return http.StatusInternalServerError
 	}
-	restClient, err := apiutil.RESTClientForGVK(gvk, restCfg, codec)
+	restClient, err := apiutil.RESTClientForGVK(gvk, false, restCfg, codec)
```

Comment:
What is this 'false'?

Reply:
@djwhatle `isUnstructured` is the newly-added param. I'm assuming we want it to be false unless we're explicitly dealing with Unstructured content.

Reply:
ACK